package.json

```diff
@@ -9,15 +9,16 @@
   "scripts": {
     "lint": "eslint .",
     "test": "npm run lint && npm run run-tests",
-    "run-tests": "lab bin/ server/ lib/ -t 80 -l -e 'development' -P spec",
+    "run-tests": "lab bin/ server/ lib/ -t 80 -l -C -e 'development' -P spec",
```

Comment:
ha! it's for "color output" - it looked prettier when i was doing tdd.  i can remove it if you want